x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterLookupJoinIT.java

```diff
@@ -449,6 +449,18 @@ public void testLookupJoinEmptyIndex() throws IOException {
             );
             assertThat(ex.getMessage(), containsString("Unknown column [v] in right side of join"));
         }
+
+        ex = expectThrows(
```

Comment:
Is this error related with the fix?

Reply:
It just surfaced with the fix. The local relation masked the fact that the lookup index was not actually of type `lookup`; now that we have to execute the join, that error starts to happen.